modules/ossm-migrating-reference-unsupported-configurations.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/reference/ossm-migrating-reference.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-migrating-reference-unsupported-configurations_{context}"]
+= Unsupported configuration fields in Service Mesh 3
+
+The following tables list {SMProduct} 2 `ServiceMeshControlPlane` configuration fields that are not supported in {SMProductName} 3. This does not necessarily mean the functionality has been removed. In some cases, such as add ons, you need to install the application separately and configure it separately.
+
+== Add on Configurations
```

Comment:
Repeat for a new module: New headings should trigger unique anchor IDs. You could also consider further modularizing this given how many sections there are.